docs/guides/cloud/branch-review.mdx

```diff
@@ -109,6 +102,12 @@ Clicking the branch label will link you directly to the appropriate [run overvie
 
 When on the review screen, you will see the Failures, Flaky, Pending, Added, and Modified tabs. Each tab will show you the specs that fall into that category. You can click on a test to view the test details.
 
+:::info
+
+If a branch PR is missing a CI run, you may also find helpful banners indicating the status and how it can be rectified.
```

Comment:
Docs should do the heavy lifting around "how to rectify". We should communicate that if a run is missing on either the merge base or feature commit of the PR, then the banner will specify which commit is missing the run and link out to it. Recording exactly one run for every commit is the best/only way to ensure complete and accurate comparisons.